### DISCUSSION TITLE
posts/install-mozilla-sops/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#218] [Comments]
# posts/install-mozilla-sops/

What is Mozilla SOPS?

https://technotim.live/posts/install-mozilla-sops/

**Comment by ometz123** [comment]
### Minor update:

The URL "https://api.github.com/repos/mozilla/sops/releases/latest" is now redirected. Therefore, you should use the following command to fetch the latest version of SOPS:

```bash
SOPS_LATEST_VERSION=$(curl -Ls "https://api.github.com/repos/mozilla/sops/releases/latest" | grep -Po '"tag_name": "v\K[0-9.]+')
```

Note the use of "-Ls" in the command for following redirects and displaying only error messages.

**Comment by mpacary** [comment]
Seems that sops repository as moved from **/mozilla/sops** to **/getsops/sops** & download path is also slightly different...

Updated first commands are:

```bash
SOPS_LATEST_VERSION=$(curl -s "https://api.github.com/repos/getsops/sops/releases/latest" | grep -Po '"tag_name": "v\K[0-9.]+')
```
```bash
curl -Lo sops.deb "https://github.com/getsops/sops/releases/download/v${SOPS_LATEST_VERSION}/sops_${SOPS_LATEST_VERSION}_amd64.deb"
```

**Reply by timothystewart6** [reply]
Thank you!  Fixed!
